src/Console/Application.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Console;
+
+use Infection\Command;
+use Infection\Config\InfectionConfig;
+use Pimple\Container;
+use Symfony\Component\Console\Application as BaseApplication;
+use Symfony\Component\Console\Formatter\OutputFormatterStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Application extends BaseApplication
+{
+    const NAME = 'Infection - PHP Mutation Testing Framework';
+    const VERSION = '@package_version@';
+    const LOGO = <<<'ASCII'
+    ____      ____          __  _
+   /  _/___  / __/__  _____/ /_(_)___  ____ 
+   / // __ \/ /_/ _ \/ ___/ __/ / __ \/ __ \
+ _/ // / / / __/  __/ /__/ /_/ / /_/ / / / /
+/___/_/ /_/_/  \___/\___/\__/_/\____/_/ /_/
+ 
+ASCII;
+
+    /**
+     * @var Container
+     */
+    private $container;
+
+    public function __construct(Container $container)
+    {
+        $this->container = $container;
+
+        parent::__construct(self::NAME, self::VERSION);
+    }
+
+    public function doRun(InputInterface $input, OutputInterface $output)
+    {
+        $this->buildDynamicDependencies($input);
+
+        $output->writeln(self::LOGO);
```

Comment:
Composer among a few other projects were putting it in `getHelp()` instead. Maybe it's just personal preferences I didn't really notice any difference in practice. Eventually it may be a better practice in case you leverage parallelism? Well whatev, if you're happy with it we can keep it, otherwise we can align otherselves to the others

Reply:
composer doesn't show logo when you execute `composer install` for example. I'd say that we can leave it as is right now